tech-debt.md

```diff
@@ -2,5 +2,88 @@
+
+### 3. Let's Talk About Tech Debt
+
+Once you have some picture of the above you should be constantly communicating your thoughts on tech debt to others:
+
+* your pod lead
+  * tech debt impacts cost and cost impact priorities
+  * any decision to accrue tech debt *has to* accept the long term cost
+* developers/software engineers
+  * you may have insights into tech debt that will impact their work (and they yours)
+  * you may be able to help them avoid tech debt in the first place (and they you)
+* software architects
+  * they have some of the best knowledge on existing tech debt
+  * they will be able to advise how to avoid tech debt in the first place
+  * they can help make your case to others that a certain piece of tech debt needs to be addressed
+  * you can help keep them up to date with the general picture of tech debt
+
+#### Where to do this?
+
+The existing communication channels are perfect for talking about tech debt. We just need to make sure we are using them.
+
+* *Stand ups* can be a great place to flag any technical debt you are facing or expecting to face in the immediate term.
+* *Sprint planning* can be used to flag any technical debt that may impact upcoming work, and can be used to plan work to address existing debt.
+* *Retrospectives* are ideal for reflecting on whether technical debt had unforeseen consequences, the overall state of tech debt and whether we need to make any changes to deal with it.
+* *Work in Progress* pull requests can be used to get feedback on changes before they are ready for formal code review. *Early feedback can go a long way to avoid technical debt in the first place.*
+* *Pull requests* are perfect for talking about tech debt in the context of a specific change. It is the PR owner's job to draw reviewers' attention to these considerations, and reviewers should ensure that PR owners are doing this correctly. PRs should outline:
+  * what existing technical debt the changes are impacted by
+  * what, if anything, the changes do to help address existing tech debt
+  * what technical debt the changes introduce or compound, and any justification for them
+  * what GitHub issues and/or Jiras have been raised to address any new debt in the future
+* *Guilds* can be used to see if people in other pods are having a similar experience with an existing piece of tech debt.
+
+### 4. Limit its influence
+
+Technical debt that is isolated has much less scope to interfere with other code. This tends to emerge naturally if code is loosely coupled and modular in nature.
+
+Say you have a module and its internal implementation is mess, but its interface is well thought out and does not change very often. This is an example of isolated technical debt. The messy internals will rarely interfere with work on the wider application. If however the interface becomes the subject of many changes, or it has a lot of bugs, it might be a good time to refactor or even just throw it out and start again.
```

Comment:
Might be worth linking in here to the facade design pattern. It's a useful practice for isolating debt like this and might be a good place to give people a bit more information.